packages/ui/src/components/SelectInputContainer/index.tsx

```diff
@@ -212,11 +208,8 @@ function SelectInputContainer<
     const handleOptionClick = useCallback(
         (valueKey: OPTION_KEY, value: OPTION) => {
             onOptionClick(valueKey, value, name);
-            if (!persistentOptionPopup) {
-                handleHideDropdown();
-            }
```

Comment:
Why remove handling of persistentOptionPopup? This should either be removed from the props or not removed at all.